### ISSUE TITLE
Unable to start tilt configurations after 11/15/2022

### ISSUE THREAD

**Issue by djcp** [tilt-dev/tilt#5974]
### Expected Behavior

`tilt up` to work and bring up a configuration.

### Current Behavior

`tilt up` fails with:

```
Error: unable to create controller manager: Get "https://127.0.0.1:37877/api?timeout=32s": x509: certificate has expired or is not yet valid: current time 2022-11-16T10:18:52-05:00 is after 2022-11-15T17:28:15Z
```
This fails on ubuntu 20.04 whether we're using `kind` locally, or against a remote `eks` k8s context.

This works fine on osx m1 monterey, but other mac users are reporting the same problem (on ventura?) in the tilt slack.

Updating the tilt client to the latest release did not fix this.

### Steps to Reproduce
1. Be on ubuntu 20.04
2. run `tilt up`
3.
4.

#### `tilt doctor` Output

```
$ tilt doctor
Tilt: v0.30.7, built 2022-08-12
System: linux-amd64
---
Docker
- Host: unix:///var/run/docker.sock
- Server Version: 20.10.21
- API Version: 1.41
- Builder: 2
- Compose Version: v1.25.0 (build unknown)
---
Kubernetes
- Env: unknown
- Context: remote-dev
- Cluster Name: remote-dev
- Namespace: djcp
- Container Runtime: docker
- Version: v1.21.14-eks-fb459a0
- Cluster Local Registry: none
---
Thanks for seeing the Tilt Doctor!
Please send the info above when filing bug reports. 💗

The info below helps us understand how you're using Tilt so we can improve,
but is not required to ask for help.
---
Analytics Settings
--> (These results reflect your personal opt in/out status and may be overridden by an `analytics_settings` call in your Tiltfile)
- User Mode: default

<snip>
```

#### About Your Use Case

This completely blocks tilt on the machines it effects. A show stopper.  Is there a way to regenerate the cert in `~/.tilt-dev/config`? One other oddity - there is no cert in `~/.tilt-dev/config` on my working osx machine, but there is on the broken `ubuntu` one.

**Comment by nicks** [comment]
for now, try deleting the certs under:

```
rm -r ~/Library/Caches/tilt-dev/certs
```

or on ubuntu:

```
rm -r ~/.cache/tilt-dev/certs
```

**Comment by nicks** [comment]
we just released tilt v0.30.12 with a fix for this as well.
